source/_posts/2018-07-11-hassio-images.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 month we are happy to announce our new images for Hass.io, based on HassOS.
+
+Note that the descriptions of new System can be get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficent operating system to run Docker like a hypervisor. It have exactly that software installed is need to run a supervisor. We have the focus also on security, there are no default passwords and we use [AppArmor] to protect application and containers on HassOS.
+
+Key features:
+- Safe and Secure updating with [Rauc] over USB or internet (OTA)
+- Use LZ4 compressed root files and parts of Memory
+- Read-Only root file system, designed to run on embedded systems
+- Dbus connected hosts services
+- Latest LT linux Kernel
```

Comment:
Maybe expand LT (I'm guessing that's long term support?). Remove the capital from Kernel